man/repart.d.xml

```diff
@@ -919,10 +919,9 @@
         for the merged partition for some other reason.</para>
 
         <para>The following fields are merged into the target definition in the specified ways:
-        <varname>Weight=</varname> and <varname>PaddingWeight=</varname> are simply overwritten;
-        <varname>SizeMinBytes=</varname> and <varname>PaddingMinBytes=</varname> use the larger of the two
-        values; <varname>SizeMaxBytes=</varname> and <varname>PaddingMaxBytes=</varname> use the smaller
-        value; and <varname>CopyFiles=</varname>, <varname>ExcludeFiles=</varname>,
+        <varname>Weight=</varname>is simply overwritten; <varname>SizeMinBytes=</varname> uses the larger of
+        the two values; <varname>SizeMaxBytes=</varname> uses the smaller value; and
+        <varname>CopyFiles=</varname>, <varname>ExcludeFiles=</varname>,
```

Comment:
Hmm, I don't find the premise in the commit message at all obvious. Let's say I want to create either ESP + large XBOOTLDR, or just a larger ESP if possible. And the padding after the large partition is supposed to be as specified in the second config, i.e. the supplement. The original design in 2cb9c68c3a898954a99a915006ea82822450e2c9 allows this, which is nice and consistent. With your change, we suddenly ignore the padding, which I don't see a good justification for.